bundledata.go

```diff
@@ -40,6 +41,17 @@ func (lbd *legacyBundleData) setBundleData(bd *BundleData) error {
 		lbd.Applications = lbd.LegacyServices
 		lbd.unmarshaledWithServices = true
 	}
+	for appName, app := range lbd.Applications {
+		// Kubernetes bundles use "scale" instead of "num_units".
+		if app.Scale > 0 && app.NumUnits == 0 {
```

Comment:
Only NumUnits is used downstream in the bundlechanges package.
I thought about naming this attribute Scale_ so that it's obvious it's internal. I'll do that.